src/database/models.ts

```diff
@@ -140,6 +146,9 @@ export default class Models {
         NotificationCollection.MESSAGES,
     );
 
+    // Puzzle
+    static PuzzleItem: Model<PuzzleItem> = getModel(PuzzleItem, Group.PUZZLE, PuzzleCollection.RUNES_AND_RIDDLES);
```

Comment:
should be `PuzzleItems`, and by this standard `RUNES_AND_RIDDLES` should be `Puzzle`